docs/src/api/class-dialog.md

```diff
@@ -80,6 +80,30 @@ with sync_playwright() as playwright:
     run(playwright)
 ```
 
+```csharp
+using Microsoft.Playwright;
+using System.Threading.Tasks;
+
+class DialogExample
+{
+    public static async Task Run()
+    {
+        using var playwright = await Playwright.CreateAsync();
+        var browser = await playwright.Chromium.LaunchAsync();
+        var page = await browser.NewPageAsync();
+
+        page.Dialog += async (_, dialog) =>
```

Comment:
Does C# support async event handlers?

Reply:
Yup - like in the example. It makes calling `async` methods further easier :-).

Reply:
We should follow up to fix this here (and maybe in other places).